__test__/index.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env jest */
+
+import whenAllSettled from '../src/.'
+
+describe('whenAllSettled', () => {
+    test('should call console.error if no rejection handler is provided', async () => {
+        const promises = []
+        promises.push(new Promise((resolve, reject) => {
+            reject(new Error('Failed Promise'))
+        }))
```

Comment:
They made `Promise.reject` and `Promise.resolve` as shorthands exactly for this usage ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#Methods)). So it could just be e.g. `const promises = [Promise.reject(new Error('Failed Promise'))]`